.dockerignore

```
@@ -60,6 +60,7 @@
 !frontend/index.html
 !frontend/public
 !frontend/src
+!frontend/vite
```

Are we sure that we always want to include the hole `vite` folder? 


```suggestion
!frontend/vite/mdilcons.js
```